index.html

```diff
@@ -0,0 +1,446 @@
+  <link rel="stylesheet" href="styles.css">
+  <title>Dashboard</title>
+</head>
+<body>
+  <div class="container">
+
+    <div class="navigation-menu">
+      <div class="logo">
+        <div class="icon">
+          <img src="./assets/menu-icons/logo-icon-svg.svg" alt="logo of the page">
+        </div>
+      </div>
+
+      <div class="menu">
+        <ul>
+          <li id="first-li">Dashboard</li>
+          <li id="second-li">Exam date</li>
+          <li id="third-li">Class Room</li>
+          <li id="foutrh-li">Notice Board</li>
+          <li id="fifth-li">Messages</li>
+          <li id="sixth-li">Results</li>
+          <li id="seventh-li">Notification
+            <div class="amount-of-notifications">
+              <span>5</span>
+            </div>
+          </li>
+          <li id="eighth-li">Settings</li>
+        </ul>
+      </div>
+
+      <div class="download">
+        <img src="./assets/menu-icons/download-img.png" alt="download image">
+        <div><a href="">Download</a> our letest apps now !</div>
+      </div>
+    </div>
+
+    <div class="main-content">
+      <header>
+        <div class="user-info">
+          <div class="user-photo">
+            <img src="./assets/header-icons/user-photo.svg" alt="">
+          </div>
+          <div class="user-name">Farhad Hossen</div>
+          <div class="more-user-info-arrow">
+            <img src="./assets/header-icons/arrow.svg" alt="more info about user">
+          </div>
+        </div>
+
+        <div class="search">
+            <input type="search" placeholder="Search">
+          <div class="notifications">
+            <img src="./assets/header-icons/notifications.png" alt="Notification button">
+          </div>
+        </div>
+      </header>
+
+      <div class="content">
+        <div class="statistics-container">
+          <div class="full-online-statics">
+            <div class="online-statics">
+
+              <div class="icon-background-border">
+                <div class="icon-background">
+                    <img src="./assets/statistics-icons/students-icon.svg" alt="students icon">
+                </div>
+              </div>
+
+              <div class="amount-of-index">
+                  <p class="index-name">Total Students</p>
+                  <h3 class="index-amount">1163</h3>
+              </div>
+
+              <div class="static-graph">
+                <img src="./assets/statistics-icons/students-graph.svg" alt="students praph">
+              </div> 
+
+              <div class="online-graph">
+                <div class="indicator-of-index">
+                  <p class="indicator-name">Attendantness:</p>
+
+                  <div class="online-display-of-the-indicator attendantness"></div>
+                  <div></div>
+                </div>
+
+                <p class="percents-of-the-indicator">80%</p>
+              </div>
+            </div>
+
+            <div class="online-statics">
+
```

Comment:
same
unnecessary line

we don't paste empty lines when we OPEN a tag

Reply:
fixed